config/routes/bops.rb

```diff
@@ -351,12 +351,12 @@
     end
   end
 
-  resources :enforcements, only: %i[index show] do
-    scope module: :enforcements do
-      resource :report, only: %i[show update] do
-        resource :check_details, only: :show
-      end
-    end
+  resources :enforcements, only: %i[index show]
+
+  scope "/:case_type/:case_id", constraints: {case_type: /enforcement|planning_application/} do
```

Comment:
We don't need the case type in the url as the case id is unique - we could just use `/cases`.